appengine-java8/tasks/app/src/main/java/com/example/task/Enqueue.java

```diff
@@ -0,0 +1,69 @@
+/* Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.task;
+
+// [START cloud_tasks_enqueue]
+import com.google.cloud.tasks.v2.AppEngineHttpRequest;
+import com.google.cloud.tasks.v2.CloudTasksClient;
+import com.google.cloud.tasks.v2.HttpMethod;
+import com.google.cloud.tasks.v2.QueueName;
+import com.google.cloud.tasks.v2.Task;
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+// The Enqueue servlet should be mapped to the "/enqueue" URL.
+// With @WebServlet annotation the webapp/WEB-INF/web.xml is no longer required.
+@WebServlet(
+    name = "TaskEnqueue",
+    description = "Enqueue a task with a key",
+    urlPatterns = "/taskqueues/enqueue")
+public class Enqueue extends HttpServlet {
+
+  protected void doPost(HttpServletRequest request, HttpServletResponse response)
+      throws ServletException, IOException {
+    String key = request.getParameter("key");
+
+    try (CloudTasksClient client = CloudTasksClient.create()) {
+      // TODO(developer): Replace these variables before running the sample.
+      String projectId = "my-project-id";
```

Comment:
I think these should be `static final` and scoped for the class.

Reply:
done.